src/Aspire.Hosting.Redis/RedisContainerImageTags.cs

```diff
@@ -7,7 +7,7 @@ internal static class RedisContainerImageTags
 {
     public const string Registry = "docker.io";
     public const string Image = "library/redis";
-    public const string Tag = "7.2";
+    public const string Tag = "7.4";
     public const string RedisCommanderRegistry = "docker.io";
     public const string RedisCommanderImage = "rediscommander/redis-commander";
```

Comment:
Seems there more recent drops of the Redis Commander image at https://github.com/joeferner/redis-commander/pkgs/container/redis-commander which is from the actual author. Perhaps we should consider changing to that registry?

Reply:
Sounds good to me, but I'd probably just separate that change so that it stays in main (9.0) only to get more bake time as this PR is going to get backported to 8.2.